test/e2e/cluster_size_autoscaling.go

```diff
@@ -116,6 +121,15 @@ func setUpAutoscaler(metric string, target float64, min, max int) {
 	expectNoError(err, "Output: "+string(out))
 }
 
+func createConsumingRCs(f *Framework, name string, count, cpu, mem int) []*ResourceConsumer {
```

Comment:
Can you add a comment what are the arguments here? Especially, is cpu & mem per replica or total.

There should be a similar comment in NewStatic(Dynamic)ResourceConsumer. It took me some time to figure out what is going on here.